test/users.test.js

```diff
@@ -0,0 +1,123 @@
+const request = require('supertest');
+const truncateDatabase = require('./setup');
+const app = require('../app');
+const { user, repeatedUser, passwordUser, missingUser } = require('../test/data/users');
+
+ beforeEach(async() => {
+   const response = await request(app)
+    .post('/users')
+    .set('Accept', 'application/json')
+    .send(repeatedUser);
+ });
```

Comment:
Be careful with these `beforeEach()` use them only inside specific use cases on `describes`